samples/sensor/thermometer/sample.yaml

```diff
@@ -3,11 +3,18 @@ sample:
 tests:
   sample.sensor.thermometer:
     tags: sensors
-    harness: sensor
     filter: dt_alias_exists("ambient-temp0")
     integration_platforms:
       - nrf52840dk/nrf52840   # mcp9700a
       - frdm_k22f             # tcn75a
       - robokit1              # ntc_thermistor
       - adi_eval_adin1110ebz  # adt7420
       - frdm_mcxn947/mcxn947/cpu0 # p3t1755
+    harness: console
+    harness_config:
+      fixture: sensor_ambient_temp
```

Comment:
Why is a fixture required?

Reply:
if the sensor is not on board but extend port.